platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -182,8 +184,12 @@ def __unstage_update(self, image_path):
         except subprocess.CalledProcessError as e:
             raise RuntimeError("Failed to unstage firmware update: {}".format(str(e)))
 
-    def __trigger_update(self):
-        cmd = self.ONIE_FW_UPDATE_CMD_UPDATE
+    def __trigger_update(self, reboot):
+
```

Comment:
@alexrallen what is the motivation for extra line?

Reply:
None, I can remove.